openbb_platform/CONTRIBUTING.md

```diff
@@ -287,145 +446,68 @@ class <ProviderName>StockEODFetcher(
-
-### The charting extension
-
-> In theory the same principles apply to any other extension.
-
-#### Add a visualization to an existing Platform command
-
-One should first ensure that the already implemented endpoint is available in the [charting router](openbb_platform/extensions/charting/openbb_charting/charting_router.py).
-
-To do so, you can run:
- `python openbb_platform/extensions/charting/openbb_charting/builder.py` - which will read all the available endpoints and add them to the charting router.
-
-Afterwards, you'll need to add the visualization to the [charting router](openbb_platform/extensions/charting/openbb_charting/charting_router.py). The convention to match the endpoint with the respective charting function is the following:
-
-- `stocks/load` -> `stocks_load`
-- `ta/ema` -> `ta_ema`
-
-When you spot the charting function on the charting router file, you can add the visualization to it.
-
-The implementation should leverage the already existing classes and methods to do so, namely:
-
-- `OpenBBFigure`
-- `OpenBBFigureTable`
-- `PlotlyTA`
-
-Note that the return of each charting function should respect the already defined return types: `Tuple[OpenBBFigure, Dict[str, Any]]`.
-
-The returned tuple contains a `OpenBBFigure` that is an interactive plotly figure that can be used in a Python interpreter, and a `Dict[str, Any]` that contains the raw data leveraged by the API.
-
-After you're done implementing the charting function, you can simply use either the Python interface or the API to get the chart. To do so, you'll only need to set the already available `chart` argument to `True`.
-
-Refer to the charting extension [documentation](openbb_platform/extensions/charting/README.md) for more information on usage.
-
-#### Using the `to_chart` OBBject method
-
-The `OBBject` is the custom OpenBB object that is returned by the Platform commands.
-It implements a set of `to_<something>` functions that enable the user to easily transform the data into a different format.
-
-The `to_chart` function should be taken as an advanced feature, as it requires the user to have a good understanding of the charting extension and the `OpenBBFigure` class.
-
-The user can use any number of `**kwargs` that will be passed to the `PlotlyTA` class in order to build custom visualizations with custom indicators and similar.
-
-Refer to the [`to_chart` implementation](openbb_platform/extensions/charting/openbb_charting/core/to_chart.py) for further details.
-
-> Note that, this method will only work to some limited extent with data that is not standardized.
-> Also, it is designed only to handle time series data.
+## Manage environment and dependencies
 
-## Environment and dependencies
-
-In order to contribute to the Platform you need to setup your environment in order to ensure a smooth development experience.
+In order to contribute to the OpenBB Platform, you need to setup your environment to ensure a smooth development experience.
```

Comment:
This should be at the very top of the page.  Environments needs to be setup before anything else.